fastlane/Fastfile

```diff
@@ -140,6 +137,29 @@ def coverage(scheme:, devices: nil, destination: nil)
     cobertura_xml: "true",
     build_directory: DERIVED_DATA_PATH
   )
+
+  # Since xchtmlreport depends on the result bundle path, we need to run the tests again
+  # AFTER xcov has run, because running it before causes issues finding the test logs in derived data
```

Comment:
#? Are there any bugs we can link to or anything like that? Obviously, I'd love not to have to run the tests twice, and if we can't avoid that today, I'd like to at least keep an eye toward the future at least.

Reply:
I couldn't find any documentation about a change in how Xcode treats the `-resultBundlePath` flag, but I think that's the root of the issue. My hypothesis is that in previous versions of Xcode the test result bundle was available in both derived data as well as the custom path, which is why it is no longer present.

I am going to try one more thing later today to see if we can eliminate the duplicate test run, and might report this Xcode behavior change upstream to the fastlane scan docs.